### ISSUE TITLE
Cannot see Full games anymore

### ISSUE THREAD

**Issue by MattiaBo** [maxgalbu/xbmc.plugin.video.nba#15]
Hi!

First of all, thanks for this plugin, way better to see the game on TV instead of the iPad!

During the last week I cannot see the full games of the Finals, while the condensed games start correctly. The error is "Failed to get a video URL. Are you logged in?"
But I am logged in and I can see the condensed videos. And I can see the full games on iPad, so I think the problem is in the plugin that it is not able to retrieve the link.

Mattia.

**Comment by shaikh3** [comment]
Same problem here.  Older games work, but the newest one is giving the same error mentioned in the post above.

**Comment by maxgalbu** [comment]
I did not change anything, the games started working without my intervention :)

I've changed the way I request the scoreboard, just in case: now I do a single request to a json with all the info: https://github.com/maxgalbu/xbmc.plugin.video.nba/commit/0d31dc3e7cb1771624abdafb28e40be2c5af43e1

**Comment by FIoannides** [comment]
I have experienced the same problem. From what I have observed, the full game starts working as soon as highlights are added, before that you will get the "Are you logged in?" error if you try to watch it.

**Comment by maxgalbu** [comment]
mm makes sense, this morning at 10am gmt+2 the hightlights weren't there.. it could be that the archived full game was still not available, but the website/apps still worked because they use the live video and can seek to the beginning...

**Comment by FIoannides** [comment]
You are right, I watched game 3 today morning (Europe time, directly from the website some hours after the game was finished) and in the afternoon after highlights were added. Indeed the feed in the morning was the live video and the feed in the afternoon was modified (removed most of pre-game, post-game and halftime shows).

**Comment by nzeppos** [comment]
Having the same problem as we speak, about 4 hours after Finals game 7th has ended. But as far as I can tell both the condensed game and the highlights are there and working, so it looks a little bit different to what you're describing above.

Also tried it with the new scoreboards update (ref. 0d31dc3) to no avail.

**Comment by thelakerlee** [comment]
Is the a fix for the failed to get video URL issue?
